lib/LWP/Simple.pm

```diff
@@ -113,37 +135,11 @@ method request_shell (RequestType $rt, Str $url, %headers = {}, Any $content?) {
         }
 
         when / 20 <[0..9]> / {
```

Comment:
`200 <= $_ <= 209`

Reply:
I'm actually tempted to make some subsets like `subset OKResponse where 200 <= * <= 209` just to make it a little neater.

Reply:
Waiting for that change...

Reply:
It wants to be done as a separate piece of work as it was like it before this and the original PR was just following that.